exercism/secret-handshake/secret-handshake.js

```diff
@@ -0,0 +1,101 @@
+module.exports = class SecretHandshake{
+  constructor(num){
+    if(!Number.isInteger(num)){
+      throw new Error('Handshake must be a number')
+    }
+    this.num = num
+    this.secret = []
+  }
+
+  commands(){
+    console.log(this.num)
+    if(this.num & 1){
```

Comment:
Are they having you chain bitwise operations here?  That's pretty nuts.  

Looks good to me.